onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -407,17 +414,47 @@ Applies to session load, initialization, etc. Default is 0.)pbdoc")
 This parameter is unused unless *enable_sequential_execution* is false.)pbdoc")
       .def_property_readonly(
           "graph_optimization_level",
-          [](const SessionOptions* options) -> uint32_t {
-            return static_cast<uint32_t>(options->graph_optimization_level);
+          [](const SessionOptions* options) -> GraphOptimizationLevel {
+            GraphOptimizationLevel retval = ORT_ENABLE_EXTENDED;
```

Comment:
set this to basic as it is the default level

Reply:
fixed